site/content/faq.md

```diff
@@ -0,0 +1,19 @@
+# Frequently Asked Questions
+
+#### *FAQ: I have an existing .NET application. Can I use the AWS Deploy Tool to deploy it to AWS?*
+Yes, the AWS Deploy Tool can be used to deploy any cloud-native .NET applications to AWS. A cloud-native .NET application is written in .NET with intent to deploy to Linux, not tied to any Windows specific technology such as Windows registry, IIS or MSMQ, and can be deployed on virtualized compute. It cannot be used to deploy .NET Framework, Desktop, Xamarin, or other applications that do not fit the “cloud-native” criteria.
```

Comment:
Since we are adding .NET Core Windows Beanstalk support should we still be calling out no Windows specific technology?

Reply:
once we add it, the PR for it should include documentation!